2024/json-ld-wg/index.html

```diff
@@ -144,97 +144,53 @@ <h1 id="title"><i class=todo>DRAFT</i> JSON-LD Working Group Charter</h1>
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
         <p>
-          JSON-LD (JavaScript Object Notation for Linked Data) is a method of encoding linked data using JSON.
-          The encoding is used used mostly for search engine optimization activities but also for applications such as
-          biomedical informatics, and representing provenance information.
+          JSON-LD (JavaScript Object Notation for Linked Data) is a widely used method of encoding Linked Data using JSON.
+          Use of JSON-LD has grown and expanded over the last decade (since it's first publication as a
+          <a href="https://www.w3.org/TR/json-ld-api1/">W3C Recommendation in 2014</a>).
+          JSON-LD remains a popular choice for ecoding <abbr title="Seach Engine Optimization">SEO</abbr>-focused
+          structured data in Web pages. JSON-LD has also become a foundational format for social media (ActivityStreams),
+          Internet of Things data (Web of Things Description), Verifiable Credentials, and Software Bill of Materials (SPDX).
+        </p>
+        <p>This growth in the applied use of JSON-LD has resulted in community interest in additional expressions of
+          JSON-LD's underlying Linked Data structure into other formats. YAML-LD has been created by the
+          <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a> for easing human
+          authoring and providing a clearer path for using Linked Data applied to popular YAML-based documents such as
+          infrastructure as code, static site front matter, and API documentation formats. Additionally, a need has
+          arised in the Verifiable Credentials space for providing a more compressed expression of JSON-LD, and work on
+          CBOR-LD was begun in the JSON-LD CG.
```

Comment:
```suggestion
          infrastructure as code, static site front matter, and API documentation formats. Additionally, a need arose
          in the Verifiable Credentials space for providing a more compressed expression of JSON-LD, and work on
          CBOR-LD was begun in the JSON-LD CG.
```

Reply:
Resolved in https://github.com/json-ld/json-ld-wg-charter/pull/11/commits/257580c5ecb4e7e44a9dd203890ce19947fb491a